content/english/articles/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-Title: Articles
-meta_title: Gnops Articles
+title: Articles
+metaTitle: Gnops Articles
```

Comment:
We can also add a `metaDescription` here for SEO if needed. By default it will be the site description -> "Official website of gnops"